nemo/collections/asr/models/msdd_models.py

```diff
@@ -18,13 +18,15 @@
 import pickle as pkl
 import tempfile
 from collections import OrderedDict
+from pathlib import Path
 from statistics import mode
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 import numpy as np
 import torch
 from hydra.utils import instantiate
 from omegaconf import DictConfig, open_dict
+from pyannote.core import Annotation
```

Comment:
Import guard unless it's a required dependency

Reply:
we already import `DiarizationErrorRate` from pyannote a line below, is an import guard necessary?

Reply:
not necessary, pyannote is part of our requirements